### ISSUE TITLE
 Segmentation fault (core dumped) when generating tf_records for R-Net 

### ISSUE THREAD

**Issue by waquey** [AITTSMD/MTCNN-Tensorflow#177]
Hello,  based on your tutorial step 10 "gen_RNet_tfrecords.py to generate tfrecords for RNet.(you should run this script four times to generate tfrecords of neg,pos,part and landmark respectively)", I've tried to generate for positive set at first. 

"get_dataset"

 item = '%s/pos_%s.txt' % (net,net)
 item = '%s/part_%s.txt' % (net,net)
 item = '%s/neg_%s.txt' % (net,net)
 item = '%s/landmark_%s_aug.txt' % (net,net)

However, it'll show the following messages after converting all images when generating tfrecords

Segmentation fault (core dumped)

Is there something I need to modify?
Thanks

**Comment by fire717** [comment]
do u changed the saved tfrecord filename? if not, the four might be replaced into one file

**Comment by waquey** [comment]
Yes, I've tried to feed them as parameter for run() in gen_RNet_tfrecords.py 

run(dir, net, output_directory, 'pos_landmark.tfrecord', ..)
run(dir, net, output_directory, 'part_landmark.tfrecord', ..)
run(dir, net, output_directory, 'neg_landmark.tfrecord', ..)
run(dir, net, output_directory, 'landmark_landmark.tfrecord', ..)

However, the tfrecords files can be generated actually

**Comment by liuyunwww** [comment]
#30

**Comment by liuyunwww** [comment]
#131

**Comment by liuyunwww** [comment]
#14

**Comment by liuyunwww** [comment]
https://github.com/AITTSMD/MTCNN-Tensorflow/issues/3
